content/source/docs/enterprise-beta/getting-started/vcs.html.md

```diff
@@ -12,29 +12,14 @@ Before you can use TFE, it needs access to the version control system (VCS) you
 
 ## About VCS Access
 
-Every workspace in TFE is associated with a VCS repository, which provides the Terraform code for that workspace. To find out which repos are available, and to access their contents, TFE needs access to your VCS service.
+Every workspace in TFE is associated with a VCS repository, which provides the Terraform code for that workspace. To find out which repos are available, access their contents, and create webhooks, TFE needs access to your VCS service.
 
 ## Configuring VCS Access
 
-TFE uses the OAuth protocol to authenticate with VCS services.
-
-~> **Important:** Even if you've used OAuth before, read the instructions carefully. Since TFE's security model treats each _organization_ as a separate OAuth application, we authenticate with OAuth's developer workflow, which is more complex than the standard user workflow.
-
-The exact steps to authenticate are different for each VCS service, but they follow this general order:
-
-On your VCS | On TFE
---|--
-Register your TFE organization as a new app. Get ID and key. | &nbsp;
-&nbsp; | Tell TFE how to reach VCS, and provide ID and key. Get callback URL.
-Provide callback URL. | &nbsp;
-&nbsp; | Request VCS access.
-Approve access request. | &nbsp;
-
-For complete instructions, see the [VCS Integrations page](../vcs/index.html).
-
-Currently, TFE cannot use other VCS services (including generic Git servers).
+Before you can create workspaces based on your repositories, you must authorize TFE to access your VCS service.
```

Comment:
This isn't entirely true :-)

You can [create workspaces without VCS via the API](http://localhost:4567/docs/enterprise-beta/api/workspaces.html#create-a-workspace-without-a-vcs-repository).

I think this is worth calling out as there is definitely a set of customers who need to provision workspaces via the API without VCS.

Reply:
RIGHT, bah, I meant to revise that and missed it. 👍